meta-coav/LICENSE

```diff
@@ -0,0 +1,54 @@
+Apache License
```

Comment:
Well, it is checking the one from the project, this is the license for the meta layer itself (they just so happen to be the same). Maybe it makes more sense when it moves to a separate repo, but it will eventually make its way into the layer. I can remove it for now.